custom_components/plugwise/climate.py

```diff
@@ -128,7 +135,7 @@ def target_temperature_low(self) -> float:
     @property
     def hvac_mode(self) -> HVACMode:
         """Return HVAC operation ie. auto, heat, heat_cool, or off mode."""
-        if (mode := self.device["mode"]) is None or mode not in self.hvac_modes:
+        if (mode := self.device.get("mode")) is None or mode not in self.hvac_modes:
```

Comment:
Isn't this is double?
The original line covers two different possibilities, the new line only one.

Reply:
The new one is down streaming from core (and also includes safeguarding)

Reply:
After double-checking, `mode` is never `None`, the backend-code makes sure of that. That's why I removed the `get()` in the past.
So, this get() should also be removed in Core.

Reply:
👍